includes/REST_API/Stories_Controller.php

```diff
@@ -140,20 +206,19 @@ public function get_item_schema() {
 		];
 
 		$schema['properties']['featured_media_url'] = [
-			'description' => __( 'URL to enqueue the image', 'web-stories' ),
+			'description' => __( 'URL for the story\'s poster image (portrait)', 'web-stories' ),
 			'type'        => 'string',
 			'format'      => 'uri',
 			'context'     => [ 'view', 'edit', 'embed' ],
 			'readonly'    => true,
 			'default'     => '',
 		];
 
-		$schema['properties']['poster_portrait_url'] = [
```

Comment:
Removed this since featured media URL was used for this anyway, it was just duplicating the same value field.